src/plugins/intel_cpu/src/emitters/snippets/x64/jit_kernel_emitter.cpp

```diff
@@ -24,7 +24,10 @@ inline static std::vector<size_t> transform_snippets_regs_to_idxs(const std::vec
     return idxs;
 }
 
+jit_snippets_call_args::jit_snippets_call_args() : amx_tile_config(new amx_tile_config_t()) {}
```

Comment:
the `amx_tile_config` field does not represent a dynamic entity, so it's perfectly fine to store it by value and initialize inplace (like all POD):

`amx_tile_config_t amx_tile_config;`

This would simplify the code a bit.

Reply:
Thanks for the suggestion! Done